model/src/main/java/com/ge/predix/acs/rest/AttributeAdapterConnection.java

```diff
@@ -32,6 +36,7 @@ public AttributeAdapterConnection(final AttributeAdapterConnection other) {
         this.uaaClientSecret = other.uaaClientSecret;
     }
 
+    @ApiModelProperty(value = "URL to a remote attribute store adapter", required = true)
```

Comment:
Then it sounds like it's a URL to attribute store itself.  How about "Adapter URL to retrieve attributes from"?